ch02/routing.js

```diff
@@ -0,0 +1,24 @@
+/* Example-1 */
```

Comment:
Are these both example 1 in the book?  I don't have it in front of me, but that seems unlikely.  I didn't intend to include all of the tiny examples in the repo, but I don't have a particular objection to it, either.  Maybe the files should be named according to the correct example number?  Like _example-2.1.js_, etc?  Thoughts?